src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable,Measurable{
```

Comment:
Ensure that the `Measurable` interface is defined and includes the `getArea()` method, as it is used in the `draw()` method. If `Measurable` is not defined, this will cause a compilation error.